pointmatcher/DataPointsFilters/utils/utils.h

```diff
@@ -41,6 +41,18 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 namespace utils {
```

Comment:
I'd choose this namespace more specific to Pointmatcher / Octree. What about `PointMatcherSupport` as in https://github.com/ethz-asl/libpointmatcher/blob/master/pointmatcher/PointMatcher.h#L78 ?

Reply:
No problem for the namespace. Should I keep the functions id the `utils.h` header or move them elsewhere ?